lib/kino/text.ex

```diff
@@ -37,6 +40,11 @@ defmodule Kino.Text do
       are merged into a single text. This is useful for streaming content.
       Defaults to `false`
 
+    * `:style` - a keyword list or map of pairs HTML styles, such as
```

Comment:
The typespec above only supports keywordlist, so perhaps let's do only that? Also HTML style is an implementation detail, so probably makes more sense to mention CSS instead.

```suggestion
    * `:style` - a keyword list of CSS attributes, such as
```